conf/messages

```diff
@@ -72,7 +72,7 @@ return_frequency.February = February, May, August and November
 return_frequency.March = March, June, September and December
 return_frequency.Monthly = Monthly
 
-confirm_return_frequency.changeLink = Change the VAT Return dates
+confirm_return_frequency.changeLink = Change the VAT return dates
```

Comment:
I think there is an error in the description of the story (my fault) and this should actually be "Return"